client/valor/schemas/filters.py

```diff
@@ -41,6 +41,47 @@
 
 @dataclass
 class Filter:
+    """
+    Filter object.
```

Comment:
```suggestion
    An object used to filter evaluations using other Valor objects (e.g., datasets, models, datums, etc.).
```

Reply:
Updated with `A data class that encapsulates filter conditions for various Valor components.`